mlflow/keras.py

```diff
@@ -82,9 +82,11 @@ def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_o
                                 'tensorflow=1.8.0'
                             ]
                         }
-    :param keras_module: Keras module to be used to save / load the model. If not provided, MLflow
-    will attempt to infer the Keras module based on the given model.
     :param mlflow_model: MLflow model config this flavor is being added to.
+    :param custom_objects: Keras custom_objects dictionary containing String:Class
+    mappings for custom layers in your Keras model.
```

Comment:
Since ``custom_objects`` is defined on the *save* method in MLflow, in contrast to the *load* method in `tf.keras` and `keras`, it would be helpful to explain what we do with these objects. Something like:
```
MLflow saves these custom layers using CloudPickle and restores them automatically when the model is loaded with :py:func:`mlflow.keras.load_model` and :py:func:`mlflow.pyfunc.load_model`.
```